modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/snapshot/incoming/IncomingSnapshotCopier.java

```diff
@@ -113,8 +147,174 @@ public SnapshotReader getReader() {
         return new IncomingSnapshotReader(snapshotMeta);
     }
 
-    @Override
-    public void close() throws IOException {
-        threadPool.shutdownNow();
+    private void start0() {
```

Comment:
Javadocs would be nice even on some private methods

Reply:
Tried to fix it.